src/pg/test/sql/select.test.js

```diff
@@ -28,6 +28,209 @@ describe('select_sql', () => {
     expectSql(selectByArgs(arg, null, options), expectedResult);
   });
 
+  test('selectByArgs_count', () => {
+    const arg = { $group: true, $all: true };
```

Comment:
When you do $group: true (no array), it will have only one child. So there is no need for any pagination like $all: true.

Reply:
Also you need to test the combination of some filter and group:true